### ISSUE TITLE
URL filter links feature redundant title content

### ISSUE THREAD

**Issue by svinkle** [Shopify/liquid#1319]
## Issue summary

The [`link_to_vendor`](https://shopify.dev/docs/themes/liquid/reference/filters/url-filters#link_to_vendor) and [`link_to_type`](https://shopify.dev/docs/themes/liquid/reference/filters/url-filters#link_to_type) URL filters generate links with `title` attributes. Depending on the screen reader or user preference settings, this may cause users to hear the same link text twice. Zoomed users may have difficulty as the tooltip that appears on hover may obscure content.

Some of the other filters also include `title` attributes with useful, but not _critical_ information.

- Related Zendesk ticket: [20130078](https://shopify.zendesk.com/agent/tickets/20130078)
- More details on why `title` is **not** always ideal can be found in the post, "[Using the HTML title attribute](https://developer.paciellogroup.com/blog/2013/01/using-the-html-title-attribute-updated/)".

#### HTML

```html
<!-- `link_to_vendor` output… -->
<a href="/collections/vendors?q=Shopify" title="Shopify">Shopify</a>

<!-- `link_to_type` output… -->
<a href="/collections/types?q=jeans" title="jeans">jeans</a>
```

## Steps to reproduce

1. Go to https://shopify.dev/docs/themes/liquid/reference/filters/url-filters#link_to_vendor with any supported browser
2. Review the `link_to_vendor` and `link_to_type` URL filter output

### Expected

- `title` attribute to not be included in the generated markup.

### Actual

- `title` attribute is present.

## Recommendation

Update each URL filter to not include a redundant `title` attribute.

Alternatively, provide a mechanism to remove the `title`, such as a third parameter as seen in [`link_to`](https://shopify.dev/docs/themes/liquid/reference/filters/url-filters#link_to).

#### HTML

```html
<!-- `link_to_vendor` output… -->
<a href="/collections/vendors?q=Shopify">Shopify</a>

<!-- `link_to_type` output… -->
<a href="/collections/types?q=jeans">jeans</a>
```

## Specifications

- **WCAG Principle:** Perceivable
- **WCAG SC:** 1.3.1 Info and Relationships
- **Severity:** Medium
- **Effort:** Low
- **Zendesk:** [20130078](https://shopify.zendesk.com/agent/tickets/20130078)

**Comment by pushrax** [comment]
This should be opened in Shopify/shopify and/or Shopify/storefront-renderer as these tags are not implemented in Shopify/liquid.

**Comment by svinkle** [comment]
New ticket: https://github.com/Shopify/shopify/issues/266102
